casper/contracts/simple_casper.v.py

```diff
@@ -561,65 +611,42 @@ def vote(vote_msg: bytes <= 1024):
 # Cannot sign two votes for same target_epoch; no surround vote.
 @public
 def slash(vote_msg_1: bytes <= 1024, vote_msg_2: bytes <= 1024):
-    # Message 1: Extract parameters
-    msg_hash_1: bytes32 = extract32(raw_call(self.MSG_HASHER, vote_msg_1, gas=self.MSG_HASHER_GAS_LIMIT, outsize=32), 0)
-    values_1 = RLPList(vote_msg_1, [int128, bytes32, int128, int128, bytes])
-    validator_index_1: int128 = values_1[0]
-    target_epoch_1: int128 = values_1[2]
-    source_epoch_1: int128 = values_1[3]
-    sig_1: bytes <= 1024 = values_1[4]
-
-    assert self.validate_signature(msg_hash_1, sig_1, validator_index_1)
-
-    # Message 2: Extract parameters
-    msg_hash_2: bytes32 = extract32(raw_call(self.MSG_HASHER, vote_msg_2, gas=self.MSG_HASHER_GAS_LIMIT, outsize=32), 0)
-    values_2 = RLPList(vote_msg_2, [int128, bytes32, int128, int128, bytes])
-    validator_index_2: int128 = values_2[0]
-    target_epoch_2: int128 = values_2[2]
-    source_epoch_2: int128 = values_2[3]
-    sig_2: bytes <= 1024 = values_2[4]
-
-    assert self.validate_signature(msg_hash_2, sig_2, validator_index_2)
-
-    assert validator_index_1 == validator_index_2
-    assert msg_hash_1 != msg_hash_2
-    assert not self.validators[validator_index_1].is_slashed
-
-    # Detect slashing
-    slashing_condition_detected: bool = False
-    if target_epoch_1 == target_epoch_2:
-        # NO DBL VOTE
-        slashing_condition_detected = True
-    elif (target_epoch_1 > target_epoch_2 and source_epoch_1 < source_epoch_2) or \
-            (target_epoch_2 > target_epoch_1 and source_epoch_2 < source_epoch_1):
-        # NO SURROUND VOTE
-        slashing_condition_detected = True
-    assert slashing_condition_detected
+    assert self.slashable(vote_msg_1, vote_msg_2)
+
+    # Extract validator_index
+    # vote messages were shown to be the same in `slashable`
```

Comment:
should be something like `# slashable garantees validator_index were same in both vote messages`